README.md

```diff
@@ -164,6 +54,8 @@ requires winpty to emulate UNIX's PTY API. The agent is a single binary
 executable and is available through the
 [GitHub releases page](https://github.com/alacritty/alacritty/releases).
```

Comment:
I suggest changing this to:

```
On Windows, Alacritty requires winpty for versions older than Windows 10
(October 2018 Update) and Microsoft's VC++ redistributable. The winpty agent
(`winpty-agent.exe`) **must** be in the same directory as the Alacritty
executable and is available through the
[GitHub releases page](https://github.com/alacritty/alacritty/releases).
```

That way we don't have things split into a million different paragraphs.